name-registry/tests/functions/identity.rs

```diff
@@ -0,0 +1,46 @@
+mod success {
+    use crate::utils::{
+        abi::{identity, register, set_identity},
+        get_contract_instance,
+    };
+    use fuels::prelude::*;
+
+    #[tokio::test]
+    async fn can_get_identity() {
```

Comment:
I would move as much of the setup to the top of the function so that I don't have to look for wherever any variable might be defined inside the middle of a test.